indico/modules/rb_new/templates/booking_list.html

```diff
@@ -0,0 +1,122 @@
+            <table class="i-table-widget">
+                <thead>
+                    <tr>
+                        <th class="room-column">{% trans %}Room{% endtrans %}</th>
+                        <th>{% trans %}Reason{% endtrans %}</th>
+                        <th>{% trans %}For whom{% endtrans %}</th>
+                        <th>{% trans %}Date{% endtrans %}</th>
+                        <th class="time-column">{% trans %}Time{% endtrans %}</th>
+                        <th class="link-column">{% trans %}Linked to{% endtrans %}</th>
+                        <th class="action-column"></th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for reservation in reservations %}
+                        <tr>
+                            <td>{{ reservation.room.full_name }}</td>
+                            <td>{{ reservation.booking_reason }}</td>
+                            <td>{{ reservation.booked_for_name }}</td>
+                            <td>
+                                {{ reservation.start_dt | format_date()}}
+                                {% if reservation.is_repeating %}
+                                    ({% trans %}recurring{% endtrans %})
+                                {% endif %}
+                            </td>
+                            <td>{{ reservation.start_dt | format_time() }} - {{ reservation.end_dt | format_time() }}</td>
+                            {% set link = reservation.link %}
+                            <td class="link-data">
+                                <span title="{{ link.object.full_title if link.link_type.name == 'session_block' else link.object.title }}">
+                                    {{ link.link_type.name.replace('_', ' ') | capitalize }}
+                                </span>
+                            </td>
+                            <td class="action-column">
+                                <a href="{{ booking_details_url(reservation) }}" target="_blank"
+                                   title="{% trans %}See details{% endtrans %}" class="icon-info action-icon"></a>
+                            </td>
+                        </tr>
+                    {% endfor %}
+                </tbody>
+            </table>
+        </div>
+    </div>
+    <script>
+        (function() {
+            'use strict';
+
+            $('#contribution, #session_block').on('change', (e) => {
+                const $target = $(e.currentTarget);
+                const $bookBtn = $target.closest('.searchable-field').find('.book-btn');
+                const linkType = {
+                    contribution: 'contribution',
+                    session_block: 'sessionBlock',
+                }[$target.attr('id')];
+                const params = {
+                    link_type: linkType,
+                    link_id: +$target.val(),
+                };
+                $bookBtn.toggleClass('disabled', !$target.val())
```

Comment:
i'd wrap the `.toggleClass` call into a new line as well. looks weird to have multiple chained calls that don't all start in the same column